lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,65 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: integer
+      )
+    end
+
+    def analyzer_version
+      Runners::VERSION
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      Results::Success.new(
+        guid: guid,
+        analyzer: analyzer,
+        issues: changes.changed_paths.map { |path| generate_issue(path) }
+      )
+    end
+
+    private
```

Comment:
[nitpick] Sorry, please add an empty line between `private` and `def` 🙏 

```suggestion
    private

```

Reply:
fix: b7743ea9d2ddb1a1fc844aef34354ff653bf52f2